src/screens/SendReceive/AddressBox.tsx

```diff
@@ -33,8 +33,8 @@ interface AddressBoxProps extends PressableProps {
 }
 
 const AddressBox = ({ addressHash, isSelected, ...props }: AddressBoxProps) => {
-  const selectAddressesAssets = useMemo(makeSelectAddressesAssets, [])
-  const assets = useAppSelector((s) => selectAddressesAssets(s, [addressHash]))
+  const selectAddressesKnownFungibleTokens = useMemo(makeSelectAddressesKnownFungibleTokens, [])
```

Comment:
Nit: what about separating the useAppSelector calls from the selector factories with a new line?

Reply:
I kinda like keeping them close to each other, they are tightly coupled. If you have a strong opinion about this I can change it, but I'd like to keep it as it is.

Reply:
Okay, very nit anyway :)